__tests__/url-utils.ts

```diff
@@ -107,7 +120,7 @@ describe('hasContentApiParamaters()', () => {
 
     expect(hasContentApiParameters(url)).toBe(false)
   })
-
+  
```

Comment:
Run `yarn format` at the end to autoformat all files (there should be also a shortcut to format the files directly in visual studio)